.azure/pipelines/jobs/default-build.yml

```diff
@@ -144,7 +144,7 @@ jobs:
   - ${{ if eq(parameters.artifacts.publish, 'true') }}:
     - task: PublishBuildArtifacts@1
       displayName: Upload artifacts
-      condition: eq(variables['system.pullrequest.isfork'], false)
+      condition: or(eq(variables['system.pullrequest.isfork'], false), eq( '${{ parameters.artifacts.force }}', 'true'))
```

Comment:
```suggestion
      condition: or(eq(variables['system.pullrequest.isfork'], false), eq(parameters.artifacts.forks, 'true'))
```

Reply:
did you mean `force` ?

Reply:
apparently I reverted one of my other changes, it states `forks`, right

Reply:
I got this, but it's possible that it was because at that time it was using `force` instead of `forks`:
`##[error]Unrecognized value: 'parameters'. Located at position 58 within expression: or(eq(variables['system.pullrequest.isfork'], false), eq(parameters.artifacts.forks, 'true'))`